packages/tecrock-simulation/src/stores/simulation-store.ts

```diff
@@ -429,6 +429,9 @@ export class SimulationStore {
   @action.bound loadCloudModel(modelId: string) {
     this.modelState = "loading";
     loadModelFromCloud(modelId, (serializedModel: ISerializedState) => {
+      if (!serializedModel) {
```

Comment:
If `serializeModel` can be undefined or null, the type of loadModelFromCloud should be changed too.

Reply:
Done in https://github.com/concord-consortium/tectonic-explorer/pull/365/commits/b1c10dc9838b17cd21b9943eecdd3984fa4edbe6